tests/integration/routes/app/category.spec.js

```diff
@@ -19,6 +19,23 @@ describe('INTEGRATION TESTS - CATEGORIES', () => {
       });
   });
 
+  describe('GET /v1/app/categories/:categoryName/posts', () => {
+    it('should return posts from category', (done) => {
+      const categoryName = defaultCategory.name;
+      request.get(`/v1/app/categories/${categoryName}/posts`)
+        .end((err, res) => {
+          const { payload } = res.body;
+
+          expect(res.statusCode).to.be.equal(200);
+          expect(payload).to.be.an('array');
+          expect(payload).to.satisfy(posts => posts.every(post => post.category === category));
+          expect(payload[0].author).to.be.an('object').that.has.all.keys('_id', 'name');
```

Comment:
To com preguiça 🤕
Eu preciso instalar o mongo na maquina e depois subir o mongo, e depois apontar pro mongo na minha máquina.
Fiquei com medo de rodar os testes no mongo da API, porque iria apagar tudo.
Tem como alguém dar uma olhada nesse teste? Não vai ser dificil de acertar, ele tem que ter o mesmo comportamento do outro endpoint.